src/Makefile.am

```
@@ -153,7 +155,7 @@ src_libt8_la_SOURCES = \
 src_libt8_la_CPPFLAGS = $(AM_CPPFLAGS) $(T8_CPPFLAGS)
 ## This is the official API versioning scheme of libtool.  Please see:
 ## Read https://www.gnu.org/software/libtool/manual/libtool.html#Versioning
-src_libt8_la_LDFLAGS = -release $(VERSION) -version-info 2:0:0
+src_libt8_la_LDFLAGS = -version-info 2:0:0
```

```suggestion
src_libt8_la_LDFLAGS = -version-info 2:0:0
```
Should this not be 1:6:1 then and change with each release?

---

or can we even use our own version macros here

---

It is unconventional to add the release version to shared libraries. What is usually appended to shared libraries is the ABI version.

Links:
https://en.wikipedia.org/wiki/Soname
https://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html
https://medium.com/@manojddesilva/managing-linux-shared-libraries-be67ae14af2a